core/controllers/async_download.py

```diff
@@ -11,27 +12,41 @@
 from core.controllers.download import download
 
 
-def async_download(
-    email_address: str,
-    file_format: str,
-    funds: list[str] | None = None,
-    organisations: list[str] | None = None,
-    regions: list[str] | None = None,
-    rp_start: str | None = None,
-    rp_end: str | None = None,
-    outcome_categories: list[str] | None = None,
+def trigger_async_download(
+    email_address,
+    file_format,
+    funds=None,
+    organisations=None,
+    regions=None,
+    rp_start=None,
+    rp_end=None,
+    outcome_categories=None,
 ):
+    """
+    Triggers the do_async_download task asynchronously.
+    """
+
+    query_params = {
+        "email_address": email_address,
+        "file_format": file_format,
+        "funds": funds,
+        "organisations": organisations,
+        "regions": regions,
+        "rp_start": rp_start,
+        "rp_end": rp_end,
+        "outcome_categories": outcome_categories,
+    }
+
+    async_download.delay(query_params)
+
+
+@shared_task(ignore_result=False)
+def async_download(query_params: dict):
```

Comment:
I think it'd be nice to keep the interface visible/explicit here. I know it's duplicative, but at least it's clear what is required

```suggestion
def async_download(
    file_format: str,
    funds: list[str] | None = None,
    organisations: list[str] | None = None,
    regions: list[str] | None = None,
    rp_start: str | None = None,
    rp_end: str | None = None,
    outcome_categories: list[str] | None = None,
):
```

Reply:
Changed as requested.